metadata.yaml

```diff
@@ -3,13 +3,13 @@ display-name: Jenkins
 docs: https://discourse.charmhub.io/t/jenkins-docs-index/4831
 summary: Jenkins Continuous Integration Server
 maintainers:
-  - launchpad.net/~canonical-is-sre
+  - IS SRE <launchpad.net/~canonical-is-sre>
```

Comment:
Maybe we should update the maintainers to launchpad.net/~canonical-is-devops

Reply:
Thank you, fixed